pkg/package-server/provider/registry.go

```diff
@@ -189,6 +189,15 @@ func (p *RegistryProvider) syncCatalogSource(obj interface{}) (syncError error)
 		Namespace: source.GetNamespace(),
```

Comment:
You commit message has misspelled words: `triggred` and`addes`. You should consider[ enabling spellcheck in VIM](https://www.ostechnix.com/use-spell-check-feature-vim-text-editor/) or whichever editor you use to create your commit messages.

Reply:
neat! did not know that's an option

Reply:
funny, i went and looked. turns out i had it on. Somehow i dont see notifiations. I'll check what was wrong with that.